### PR TITLE
feat(toolkit): Upgrade amazon-states-language-service to 1.13.0

### PR DESCRIPTION
**Pull request by elvin-hwang**
Upgrading amazon-states-language-service from 1.11.0 to 1.13.0

### CODE REVIEW

File: packages/toolkit/.changes/next-release/Feature-76bb396d-6268-4d24-9373-4c7c0e91cbc4.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Step Functions: Upgrade amazon-states-language-service to 1.13. This new version adds support for JSONata and Variables."
```
Comment (justinmk3) [inline packages/toolkit/.changes/next-release/Feature-76bb396d-6268-4d24-9373-4c7c0e91cbc4.json:3]: is "JSONata" a typo?

**Comment by justinmk3** [comment]
which issues are related?

File: packages/toolkit/.changes/next-release/Feature-76bb396d-6268-4d24-9373-4c7c0e91cbc4.json (hunk 1 above)
Comment (elvin-hwang) [reply packages/toolkit/.changes/next-release/Feature-76bb396d-6268-4d24-9373-4c7c0e91cbc4.json:3]: Not a typo, it's a new json language feature that we've incorporated into StepFunctions service
https://aws.amazon.com/blogs/compute/simplifying-developer-experience-with-variables-and-jsonata-in-aws-step-functions/

**Comment by elvin-hwang** [comment]
> which issues are related?

https://github.com/aws/aws-toolkit-vscode/issues/6118
https://github.com/aws/amazon-states-language-service/pull/159
https://github.com/aws/amazon-states-language-service/pull/160

File: packages/toolkit/.changes/next-release/Feature-76bb396d-6268-4d24-9373-4c7c0e91cbc4.json (hunk 1 above)
Comment (justinmk3) [inline packages/toolkit/.changes/next-release/Feature-76bb396d-6268-4d24-9373-4c7c0e91cbc4.json:3]:
```suggestion
	"description": "Step Functions: Upgrade amazon-states-language-service to 1.13. This new version adds support for [JSONata](https://aws.amazon.com/blogs/compute/simplifying-developer-experience-with-variables-and-jsonata-in-aws-step-functions/) and Variables."
```

File: packages/toolkit/.changes/next-release/Feature-76bb396d-6268-4d24-9373-4c7c0e91cbc4.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Step Functions: Upgrade amazon-states-language-service to 1.13. This new version adds support for [JSONata](https://aws.amazon.com/blogs/compute/simplifying-developer-experience-with-variables-and-jsonata-in-aws-step-functions/) and Variables."
```
Comment (justinmk3) [inline packages/toolkit/.changes/next-release/Feature-76bb396d-6268-4d24-9373-4c7c0e91cbc4.json:3]:
```suggestion
	"description": "Step Functions: Upgrade amazon-states-language-service to 1.13. This new version adds support for [JSONata and Variables](https://aws.amazon.com/blogs/compute/simplifying-developer-experience-with-variables-and-jsonata-in-aws-step-functions/)."
```

**Comment by justinmk3** [comment]
Integ tests are failing after this PR

```
1 failing
  1) stepFunctions ASL LSP
       init:

      AssertionError [ERR_ASSERTION]: Expected values to be strictly deep-equal:
+ actual - expected ... Lines skipped

  [
    'Comment',
+   'QueryLanguage',
    'StartAt',
...
    'TimeoutSeconds',
    'Version'
  ]
      + expected - actual

       [
         "Comment"
      -  "QueryLanguage"
         "StartAt"
         "States"
         "TimeoutSeconds"
         "Version"
      
      at Context.<anonymous> (/codebuild/output/src2587939618/src/github.com/aws/aws-toolkit-vscode/packages/core/src/testInteg/stepFunctions/init.test.ts:36:16)
```

**Pull request merged**